tests/integration/inbox.bats

```diff
@@ -43,7 +43,7 @@ function teardown() {
     do
 	t=$d/$(uuidgen)
 	touch ${TESTFILES}/$t # some empty files
-	TESTFILES_NAMES+=( "$t" )
+	TESTFILES_NAMES+=( "/$t" )
```

Comment:
What is this slash `/` before `$t` for? Is it path separator? Seems should not be included in the filename.